staticfiles/templates/registration/registration_successful.html

```diff
@@ -1,18 +1,18 @@
-{% extends "base.html" %}
+<!--{% extends "base.html" %}-->
 <!DOCTYPE html>
 <html>
 <head lang="en">
     <meta charset="UTF-8">
 
-    {% block title %}
+   <!-- {% block title %}-->
     <title>Registration Successful</title>
-    {% endblock title %}
+    <!--{% endblock title %}-->
 </head>
 <body>
 
-{% block main %}
+<!--{% block main %}-->
     <h3 style="padding-top: 60px" class="text-center">Congratulations, you have successfully signed up. <br>Please verify your account by clicking the link sent to the provided email.<br>
         <p style="padding-top: 10px"><a  href="/login">Sign In</a></p></h3>
-{% endblock main %}
+<!--{% endblock main %}-->
```

Comment:
Jinja is not needed. Only Html file has to be retrieved. Shall I remove the comments?